apps/condo/domains/common/components/Comments/index.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useRef } from 'react'
+import styled from '@emotion/styled'
+import { useIntl } from '@core/next/intl'
+import { Empty, Typography } from 'antd'
+import { Comment } from './Comment'
+import { CommentForm } from './CommentForm'
+
+export type TComment = {
+    id: string,
+    content: string,
+    user: {
+        id: string,
+        name: string,
+    },
+    createdAt: string,
+    updatedAt: string,
+    deletedAt: string,
+}
+
+const Container = styled.aside`
+  background: #F5F5F5;
+  border-radius: 8px;
+  height: calc(100vh - 120px);
+
+  > * {
+    padding: 24px
+  }
+
+  display: flex;
+  flex-flow: column nowrap;
+  align-content: space-between;
+`
+const Head = styled.div`
+  border-bottom: solid thin #D9D9D9;
```

Comment:
should we extract the color?

Reply:
Can use **lightGrey** from `@condo/domains/common/constants/style.js`

Reply:
Refactored in `41e9b707`